src/app/api/util/stringPixels.ts

```diff
@@ -0,0 +1,13 @@
+export class StringPixels {
```

Comment:
the file name should be something like string-pixel-util.ts or string-pixel.util.ts